Sources/RoutingKit/PathComponent.swift

```diff
@@ -54,6 +54,13 @@ public enum PathComponent: ExpressibleByStringLiteral, CustomStringConvertible {
     }
 }
 
+extension String {
+    /// Converts a string into `[PathComponent]`.
+    public var pathComponents: [PathComponent] {
+        return self.split(separator: "/").map { .constant(.init($0)) }
```

Comment:
I think we should use the PathComponent's string initializer here instead of passing directly to constant. Otherwise we will not correctly parse strings like:
```swift
"/todos/:id"
```

Reply:
I agree, it's fixed.